app/scripts/controllers/mmi-controller.test.ts

```diff
@@ -509,7 +508,7 @@ describe('MMIController', function () {
       CUSTODIAN_TYPES['CUSTODIAN-TYPE'] = {
         keyringClass: { type: 'mock-keyring-class' },
       };
-      mmiController.messenger.call = jest
+      mmiController.messagingSystem.call = jest
```

Comment:
Hmm, this doesn't seem right. The messenger should not be accessible outside of the controller.

Can we use the `controllerMessenger` instead?